includes/class-sensei-grading.php

```diff
@@ -1275,6 +1310,57 @@ public function get_courses_average_grade() {
 
 		return doubleval( $result->courses_average );
 	}
+
+	/**
+	 * Get the average grade of the courses.
+	 *
+	 * @since 4.5.0
+	 * @access public
+	 *
+	 * @param array $courses_ids Courses ids to filter by.
+	 * @return double Average grade of all courses.
+	 */
+	public function get_courses_average_grade_filter_courses( array $courses_ids ) {
```

Comment:
Same here.

Reply:
https://github.com/Automattic/sensei/pull/5091/commits/8762130d9dbcdb3ec90438cec1da1d77d43d0801

Reply:
Yeah, sorry for not explaining it better. Correct, you could move this method and deprecate the `Sensei_Grading::get_courses_average_grade` method.